src/prognostic_equations/forcing/external_forcing.jl

```diff
@@ -181,6 +180,12 @@ function external_forcing_tendency!(Yₜ, Y, p, t, ::GCMForcing)
         Val{:first_order}(),
     )
     # <-- subsidence
+    # needed to address top boundary condition for forcings. Otherwise upper portion of domain is anomalously warm
```

Comment:
```suggestion
    # needed to address top boundary condition for forcings. Otherwise upper portion of domain is anomalously cold
```
(it's anomalously warm with a different boundary condition, which I didn't merge.)